torch/mps/__init__.py

```diff
@@ -0,0 +1,88 @@
+r"""
+This unit enables an interface for accessing MPS backend in python
```

Comment:
Any reason for this new submodule instead of using the torch.backends.mps one?

Reply:
Yes, we want parity with CUDA and other modules. We should be consistent with naming the submodule namespaces.
Example, when CUDA synchronizes like `torch.cuda.synchronize()`, the user must be able to intuitively just replace `cuda` with `mps` to get the same operation with `torch.mps.synchronize()`.